packages/web/src/pages/saved-page/components/desktop/AddPlaylistCard.tsx

```diff
@@ -0,0 +1,3 @@
+export const AddPlaylistCard = () => {
```

Comment:
is this needed?

Reply:
no good call, just going to keep the add-playlist-card inline for now